packages/terra-clinical-data-grid/docs/README.md

```diff
@@ -59,10 +59,11 @@ Rows are provided to sections to be rendered by the DataGrid. The order in which
 |Name|Type|Is Required|Default Value|Description|
 |---|---|---|---|---|
 |**id**|string|required|none|An identifier for the row. This identifier should be unique across the set of rows provided to a given section.|
-|**ariaLabel**|string|optional|A string identifier used to describe the row contents. This value will be used to contruct additional labels for internal controls (e.g. row selection cells).|
+|**ariaLabel**|string|optional|none|A string identifier used to describe the row contents. This value will be used to contruct additional labels for internal controls (e.g. row selection cells).|
 |**isSelectable**|bool|optional|none|A boolean indicating whether or not the row is selectable.|
 |**isSelected**|bool|optional|none|A boolean indicating whether or not the row should render as selected.|
 |**cells**|array|optional|[]|An array of cell objects that define the content to be rendered in the row. The order in which the cells are rendered is determined by the order of the `pinnedColumns` and `overflowColumns` props.|
+|**height**|string|optional|none|The height for this row. This overrides the `DataGrid` `rowHeight` property. Any valid CSS height value is accepted (i.e. 50px, 3rem, etc.)|
```

Comment:
In light of new better-defined accessibility requirements, we should call out `rem` units as the preferred unit.

(If there are any px's hanging out in the examples, we could update those too).

Reply:
Done in a7e965333ae07c990b02a5592a5a83e0bd0bcdda and 4e29385d3a5e8e36195ea8e9ecf2179b984b874a